src/Filters/DateFilter.jsx

```diff
@@ -9,11 +9,12 @@ import moment from 'moment'
 import PropTypes from 'prop-types'
 import React from 'react'
 import { shortFormat } from '../utils'
-import Warning from '../Warning'
 
 const WARN_SERIES_BREAK = 'In April 2017 we made changes to consumer' +
   ' complaint Product fields. This may result in discrepancies in the' +
-  ' display of visualizations';
+  ' display of visualizations ';
+
+const LEARN_SERIES_BREAK = 'http://files.consumerfinance.gov/f/documents/201704_cfpb_Summary_of_Product_and_Sub-product_Changes.pdf';
```

Comment:
Does this lint?

Reply:
That's weird. my linter isn't picking it up, but my IDE obviously shows it as longer than 80 chars.

same for line 45 and 63 below

Reply:
![](https://i.imgflip.com/3w5p3b.gif)